README.md

```diff
@@ -149,7 +147,7 @@ complete("The given entity wasn't found").notFound();
 
 ##### End processing
 
-Once the status is set, you can end the processing of the dataLink request with either `done` or `next`.  Most requests should normally end with `next`, which will continue the Kinvey request pipeline with Business Logic.  `done` will return the response that was set in the DataLink, and end request processing without executing any further business logic.
+Once the status is set, you can end the processing of the handler request with either `done` or `next`.  Most requests should normally end with `next`, which will continue the Kinvey request pipeline.  `done` will return the response that was set in the completion handler, and end request processing without executing any further functions.
```

Comment:
No, it's the request that came into the handler. It's not the kinvey request because that request could have been altered by any # of things before arriving here.